dom.bs

```diff
@@ -2195,6 +2195,11 @@ steps:</p>
  <li><p>If the <i>open flag</i> is set and <var>shadow</var>'s <a for=ShadowRoot>mode</a> is
  <em>not</em> "<code>open</code>", then return null.</p></li>
 
+ <li><p>If the shadow's <a for=ShadowRoot>slot assignment</a> is set to "<code>manual</code>",
+ return the associated <a>slot</a> in shadow where the <var>slottable</var> is
+ <a href="https://html.spec.whatwg.org/multipage/scripting.html#dom-slot-assign"><code>assigned</code></a>,
```

Comment:
This is too much action-at-a-distance. A `slot` probably needs to keep track of what was manually assigned to it in some "associated" thingie.

Reply:
Thanks Anne. I've update this portion to be more of an algorithm with internal variables, manuallyAssignedNodes and nodeSlotMap, to keep track of the association between a manually assigned node to its slot.